PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,7 @@
 
 - [ ] PR reference the relevant issue (e.g. `Fixes #007`)
 - [ ] if UI changes, a screenshot has been added to the PR
+- [ ] if method added to `xo-server` API, the corresponding test has been added to [`xo-server-test`](https://github.com/vatesfr/xen-orchestra/tree/master/packages/xo-server-test)
```

Comment:
IMHO, it should be more general, the tests should be updated for any xo-server's API change.